tests/wicked/advanced_sut.pm

```diff
@@ -40,8 +42,20 @@ sub run {
 
     # Create mutex to unlock REF
     mutex_create("test_1_ready");
+    if (!$ret) {
+        record_info("Can't ping IP $parallel_ip_in_tunnel", result => 'fail');
```

Comment:
We could add a pre-string to this message to identify easily in the logs what failed. Example:
record_info("[Test-1] Can't ping IP $parallel_ip_in_tunnel", result => 'fail');

Reply:
will do that